migrate.py

```diff
@@ -140,6 +142,8 @@ def main():
 
     results_file = os.path.join(LOG_DIR, 'lookup_{}.csv'.format(timestamp_suffix))
     logging.info("Saving Lookup Table to {}".format(results_file))
+    czo_hs_id_lookup_df["elapsed_time"] = czo_hs_id_lookup_df["elapsed_time"].\
+        apply(lambda sec: "{:.0f} sec | {:.2f} min".format(sec, sec / 60))
     czo_hs_id_lookup_df.to_csv(results_file, encoding='utf-8', index=False)
```

Comment:
@cuahsimarko Good question! I did save literal strings into the df initially, then I realized putting real numbers would allow us to do some calculations (like sum) or sortings if we want to later...